### ISSUE TITLE
Semantic features for singular "they"?

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#371]
As discussed at UniversalDependencies/docs#517, the morphological features don't distinguish different uses of "they", "their", etc., which are always `Number=Plur` and unmarked for gender. As the morphosyntactic behavior is consistently plural, this is probably the right approach.

One option to consider is marking the semantically exceptional ones with MISC features: `SemGender=Com|SemNumber=Sing`. (`Com` = Common gender, i.e., for persons but not making a masc/fem distinction).

However, this will be a challenge to implement without coreference annotations. Often the antecedent of singular "they" is an indefinite pronoun, but not necessarily (e.g., "no person who breaks their leg", "the last person to break their leg", etc.).

**Comment by nschneid** [comment]
I have manually reviewed the ~80 instances where "they" occurs in the same sentence as an indefinite pronoun and marked the ones that I thought were semantically singular (not generic "they" or "they" referring to people who comprise a business).

Pushed to a branch above. But I'm not sure it makes sense to pursue this before adding general coref.

**Comment by nschneid** [comment]
We should consider the case of "you" as well: GUM has a semantic number distinction, while in EWT (following [these docs](https://universaldependencies.org/en/pos/PRON.html)) there is no number feature.

It can be considered to be number syncretism, where both values of the feature are available but not distinguished grammatically or lexically. But the choice of singular vs. plural requires a lot of interpretation for which the context in the text may be insufficient (e.g. in a discussion forum, "you" might be taken to refer to a single reader, or all the previous commenters in the thread).

If we adopt the more semantic definition of the `Number` feature for "you", it could make sense to do so for "they" as well.

Another strategy could be to let subject-verb agreement govern the features. Since a singular-agremeent verb can never have "you" or "they" as its subject, that would be a basis for excluding `Number=Sing` regardless of semantics. In this case, it would make sense to have a MISC feature for storing the semantic value.

**Comment by amir-zeldes** [comment]
> the choice of singular vs. plural requires a lot of interpretation

This is true of a variety of things we distinguish, even as ubiquitous as VBD vs. VBN for "walked" when context is limited

> let subject-verb agreement govern the features

There are also agreement considerations in anaphora. Only plural "you" is compatible with "Kim and Yun" as a coordinate NP antecedent, for example. In GUM the number is based on the utterance level addressee field, where available. It's not perfect, but it does distinguish a number of genuine singuar vs. plural "you"s, and can be interesting for researchers. What's more, English now has things like "you all/guys/.." etc., which are explicitly plural.

> a singular-agremeent verb can never have "you" or "they" as its subject, that would be a basis for excluding Number=Sing regardless of semantics

Actually I've been hearing "they ... themsel*f*", indicating grammatical awareness of singular they, so there is some morphosyntactic justification for it even for "they".
